README.md

```diff
@@ -232,33 +232,40 @@ Here's how you can introduce a new environment variable to the code base:
 
 ### Make a release
 
-Creating a release involves three steps: creating a new version tag and GitHub Release, creating and distributing a new Android build, and creating and distributing a new iOS build.
+Creating a release involves three steps:
 
-#### Create Version Tag and GitHub Release
+- Increment the build number and optionally update the version number
+- Create and distribute a new Android build
+- Create and distribute a new iOS build
 
-1. Ensure you are on the main branch and have the latest changes.
-   ```shell
-   git checkout main && git pull
-   ```
-2. Run a test build. Ensure that this command completes successfully and leaves _no local changes_. If there are local changes, commit them separately before proceeding.
-   ```shell
-   ionic capacitor sync --prod
-   ```
-3. Decide whether the new version will be a patch, minor, or major version. Then, run the following commands to create a new version commit and tag.
+#### Incrementing build and version numbers
 
-   > [!NOTE]
-   > During the beta testing period, use only minor or patch versions.
+**Decide**: Are you releasing a new **build** or a new **version**?
 
-   ```shell
-   npm version patch  # (or "minor" or "major")
-   git push && git push --tags
-   ```
+<details>
+<summary>How do I choose?</summary>
+
+- A build release is a new **testing** release of the app. It may be thought of as a release candidate. It implies a new `buildNumber` in `package.json`. This corresponds to a new "build number" in [iOS terms](https://help.apple.com/xcode/mac/current/#/devba7f53ad4) or "versionCode" in [Android terms](https://developer.android.com/studio/publish/versioning). New builds do not go through Apple's approval process when released to TestFlight.
```

Comment:
```suggestion
- A build release is a new **testing** release of the app. It may be thought of as a release candidate. It implies a new `buildNumber` in `package.json`. This corresponds to a new "build number" in [iOS terms](https://help.apple.com/xcode/mac/current/#/devba7f53ad4) or "versionCode" in [Android terms](https://developer.android.com/studio/publish/versioning). New builds (of already-released _versions_) do not go through Apple's approval process when released to TestFlight.
```